spec/std/isa/ext/Shcounterenw.yaml

```diff
@@ -18,7 +18,12 @@ versions:
     state: ratified
     ratification_date: 2023-08
     url: https://drive.google.com/file/d/1KcjgbLM5L1ZKY8934aJl8aQwGlMz6Cbo/view?usp=drive_link
-    param_constraints:
-      HCOUNTENABLE_EN:
-        extra_validation: |
-          HPM_COUNTER_EN.each_with_index { |hpm_exists, idx| assert(!hpm_exists || HCOUNTENABLE_EN[idx]) }
+requirements:
+  idl(): |
+    -> implemented?(ExtensionName::H);
+
+    for (U32 i = 3; i < 32; i++){
+      HPM_COUNTER_EN[i] -> HCOUNTENABLE_EN[i];
+    }
```

Comment:
OK. This might be the poster child for `idl()`. I'm still tempted to push for YAML, but won't. ;-)